### PR TITLE
[Docs] Remove goal from design.md

### PR DESCRIPTION
**Pull request by ErikSchierboom**

### CODE REVIEW

**Comment by SaschaMann** [comment]
Could you explain the reason behind this change?

**Comment by ErikSchierboom** [comment]
Sorry about that. The reason for this change is that the goal doesn't offer any real information that isn't already somewhere else in the document. As such, we'd like to remove it from the `design.md` document to make contributing easier.

File: languages/julia/exercises/concept/functions-introduction/.meta/design.md [hunk 1]
```diff
@@ -1,7 +1,3 @@
-## Goal
-
-The goal of this exercise is to teach the student the fundamentals of the Concept of functions in Julia, so that they are able to understand the basic structure of Exercism exercises.
```
Comment (SaschaMann) [inline languages/julia/exercises/concept/functions-introduction/.meta/design.md:3]: > The reason for this change is that the goal doesn't offer any real information that isn't already somewhere else in the document.

Where should this information go now?

Comment (ErikSchierboom) [reply languages/julia/exercises/concept/functions-introduction/.meta/design.md:3]: I'm fine with reverting it here, where there _is_ some additional information. I just don't want it listed if there is no additional useful information. Is that okay?

**Comment by neenjaw** [comment]
LGTM :+1: for elixir

File: languages/julia/exercises/concept/functions-introduction/.meta/design.md (hunk 1 above)
Comment (SaschaMann) [reply languages/julia/exercises/concept/functions-introduction/.meta/design.md:3]: Sounds good!

**Comment by ErikSchierboom** [comment]
I've changed this a bit to only include C#/F#/Elixir for now. Any other tracks can do this too if they like this.

**Pull request merged**